sql/core/src/test/scala/org/apache/spark/sql/test/SQLTestData.scala

```diff
@@ -335,4 +343,30 @@ private[sql] object SQLTestData {
   case class ComplexData(m: Map[String, Int], s: TestData, a: Seq[Int], b: Boolean)
   case class CourseSales(course: String, year: Int, earnings: Double)
   case class TrainingSales(training: String, sales: CourseSales)
+  case class IntervalData(data: CalendarInterval)
+//  case class TestUDT() extends UserDefinedType[IExampleBaseType] {
+//
+//    override def sqlType: StructType = {
+//      StructType(Seq(
+//        StructField("intfield", IntegerType, nullable = false)))
+//    }
+//
+//    override def serialize(obj: IExampleBaseType): InternalRow = {
+//      val row = new GenericInternalRow(1)
+//      row.setInt(0, obj.field)
+//      row
+//    }
+//
+//    override def deserialize(datum: Any): IExampleBaseType = {
+//      datum match {
+//        case row: InternalRow =>
+//          require(row.numFields == 1,
+//            "TestUDT requires row with length == 1")
+//          val field = row.getInt(0)
+//          new ExampleBaseClass(field)
+//      }
+//    }
+//
+//    override def userClass: Class[IExampleBaseType] = classOf[IExampleBaseType]
+//  }
```

Comment:
Changes in this file are not related to this PR?

Reply:
Fixed

Reply:
Could you revert the other changes in the file?